### DISCUSSION TITLE
Calendar:about disabled logic

### DISCUSSION THREAD

**Discussion by kl-nevermore** [primefaces/community#667] [PrimeReact] [answered]
### But year and month have attributes `data-p-disabled`, this value never changed by `props.disabledDates` or `props.disabledDays`
https://codesandbox.io/p/sandbox/primereact-test-forked-6y4y7n?file=%2Fsrc%2Findex.js
### monthPickerValues  `data-p-disabled` always true
https://github.com/primefaces/primereact/blob/master/components/lib/calendar/Calendar.js#L3654
### yearPickerValues `data-p-disabled` always false
https://github.com/primefaces/primereact/blob/master/components/lib/calendar/Calendar.js#L3697

###  is this an issue or a feature? if it will be fixed, Is it a good choice to accept a new prop like example
```tsx
disabledDate?: (year: number, month: number, day: number) => boolean;

<Calendar
    value={date}
    onChange={(e) => setDate(e.value)}
    disabledDate={(y, m, d) => {
        return y > 1999 && m > 1;
    }}
/>
```

**Comment by melloware** [comment, marked as answer]
I would say its an enhancement that could be added.
